statementoffaith.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+---
+
+<section class="jumbotron jumbo-height" style="background-image: url('//crds-media.imgix.net//23IWaWUAICoAUG4Jassq40/24e97abb4beedcc1e41c334e06d4305e/crossroads-church-old2.png'); height: 455px;">
```

Comment:
What is `jumbo-height` doing?

Reply:
ah, thanks for catching that. It was an old class not being used anymore.